actionmailer/lib/action_mailer/base.rb

```diff
@@ -470,6 +470,7 @@ class Base < AbstractController::Base
       content_type: "text/plain",
       parts_order:  [ "text/plain", "text/enriched", "text/html" ]
     }.freeze
+    class_attribute :interceptors, default: ::Mail.class_variable_get(:@@delivery_interceptors)
```

Comment:
I don't think this should be a `class_attribute`.  If people can set interceptors on `AM::Base`, what would it do?

Reply:
I guess it's not necessary to set it as a `class_attribute`, instead I've added it as a method.

Besides tests, it's useful to use `ActionMailer::Base.interceptors` in the initializers of rails application to conditionally register or unregister interceptors based on the environment.

It's useful in doing something like this:

```ruby
if ActionMailer::Base.interceptors.include? ProductionEmailPrefixer
  ActionMailer::Base.register_interceptor ProductionEmailsInterceptors
  ActionMailer::Base.register_interceptor ProdBlockedEmailsInterceptors
elsif ActionMailer::Base.interceptors.include? StagingEmailPrefixer
  ActionMailer::Base.register_interceptor StagingEmailsInterceptors
  ActionMailer::Base.register_interceptors StagingBlockedEmailsInterceptors
end
```